### PR TITLE
Add UserEntity related tests to AppResolver

### PR DESCRIPTION
**Pull request by GalCegla**
-Add should create an app test.
-Add should get a pending change test.

### CODE REVIEW

File: packages/amplication-server/src/core/app/app.resolver.spec.ts [hunk 1]
```diff
@@ -369,9 +386,9 @@ const findManyEnvironmentsMock = jest.fn(() => {
   return [EXAMPLE_ENVIRONMENT];
 });
 
-const mockCanActivate = jest.fn(() => true);
+const mockCanActivate = jest.fn(mockGqlAuthGuardCanActivate(EXAMPLE_USER));
 
-describe('AppResolver', () => {
+describe('AppService', () => {
```
Comment (iddan) [inline packages/amplication-server/src/core/app/app.resolver.spec.ts:391]: AppResolver was the correct one

**Review by iddan — COMMENTED** [review]
Great job

File: packages/amplication-server/src/core/app/app.resolver.spec.ts (hunk 1 above)
Comment (GalCegla) [reply packages/amplication-server/src/core/app/app.resolver.spec.ts:391]: Weird, I probably branched from old master!

Comment (iddan) [reply packages/amplication-server/src/core/app/app.resolver.spec.ts:391]: But you haven't fixed it 😢

**Pull request merged**